examples/cloudfunctions-stackdriver-intro/main.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/python3
```

Comment:
You're Inconsistently using shebang on python files. 
Either use it everywhere or nowhere.
also this is a dangerous choice of shebang ([context](https://stackoverflow.com/questions/6908143/should-i-put-shebang-in-python-scripts-and-what-form-should-it-take))